CMakeLists.txt

```diff
@@ -155,7 +165,7 @@ if (MINGW OR APPLE)
       SET(Mscore_SHARE_NAME    "mscore.app/")
   endif(MINGW)
 else (MINGW OR APPLE)
-      SET(Mscore_INSTALL_NAME  "mscore-${MUSESCORE_VERSION}/")
+      SET(Mscore_INSTALL_NAME  "mscore${CMAKE_INSTALL_SUFFIX}-${MUSESCORE_VERSION}/")
```

Comment:
I tried CMAKE_EXECUTABLE_SUFFIX but I couldn't get it to do anything so instead I renamed to MSCORE_INSTALL_SUFFIX to avoid the CMAKE namespace.